### DISCUSSION TITLE
Hosting own peer server with Coturn

### DISCUSSION THREAD

**Discussion by SujithManjavana** [peers/peerjs#1214] [Q&A]
I'm planning to host my own `peer server` with `COTURN`. I don't know which machine type should I choose. I'm expecting around 100-300 calls at a time. Do I need a dedicated CPU plan for this? Could somebody please share a VPS configuration for this?

**Comment by jonasgloning** [comment]
I recommend asking the Coturn team directly. Maybe they can give specific estimates for your use case. Some general suggestions from me:

- Most WebRTC traffic doesn’t need a TURN server at all.
- Tunneling calls isn’t very CPU intensive; you probably don’t need dedicated CPUs.
- It needs even less RAM.
- Scaling is linear. You can add as many small instances as you want and scale up and down with demand. If in doubt, start oversized and remove servers until you reach a reasonable utilization.
- Coturn runs great on ARM – some providers offer these instances with a much better price/performance ratio.
- Put your server where your users are.

The public instances mostly run FreeBSD on Oracle Clouds VM.Standard.A1.Flex instances with one or two cores. 

Would you post a short field report after successful deployment for future reference?
